scripts/versioning.gradle

```diff
@@ -1,5 +1,7 @@
 task printVersionName() {
-    println(getProductionVersionName() + " (" + getProductionVersionCode() + ")")
+    doLast {
+        println(getProductionVersionName() + " (" + getProductionVersionCode() + ")")
```

Comment:
add info during printing what it is :)

Reply:
Do you mean what library is it? Does not matter, you can only run a task per project